lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -485,11 +487,12 @@ addIndices = mapM_ (`rawExecute` [])
 
 delegationToText :: W.WalletDelegation W.PoolId -> Maybe Text
 delegationToText W.NotDelegating = Nothing
-delegationToText (W.Delegating pool) = Just (W.getPoolId pool)
+delegationToText (W.Delegating pool) = Just $ toText pool
 
 delegationFromText :: Maybe Text -> W.WalletDelegation W.PoolId
 delegationFromText Nothing = W.NotDelegating
-delegationFromText (Just pool) = W.Delegating (W.PoolId pool)
+delegationFromText (Just pool) =
+    W.Delegating (fromRight (W.PoolVRFPubKey "invalid id pool") (fromText pool))
```

Comment:
this one default value is something I am not sure how to set

Reply:
In this case there is DB-corruption, or a manually edited DB?

If we were to store it as binary in the DB instead of hex, we wouldn't have to provide a dummy value. 

Arguably we would have the same amount of "invalid PoolIds" (just not mapped to the same dummy-value). But I think I'd still feel inclined to store it as binary, as the wallet will be interested in the raw binary when creating delegation certificates, and we'd have less conversions.

Maybe @KtorZ or @rvl has an opinion?